src/components/main_box/MainBox.tsx

```diff
@@ -44,16 +44,18 @@ class MainBox extends React.Component {
     const parentClass = 'MainBox';
     return (
       <div className={parentClass}>
-        <Section name="home" fullHeight={true} parentClass={parentClass}>
-          <TextFetcher id="hero-text" texts={this.props.texts}>
-            <Home text={emptyText}/>
-          </TextFetcher>
-        </Section>
-        <Section name="benefits" parentClass={parentClass}>
-          <TextFetcher id="benefits" texts={this.props.texts}>
-            <Benefits text={emptyText} pictures={this.props.pictures}/>
-          </TextFetcher>
-        </Section>
+        <div className="MainBox__container">
```

Comment:
![screenshot_20180607-091310](https://user-images.githubusercontent.com/1012055/41084298-08910bf0-6a34-11e8-9acb-f30d84549d02.png)

I think this affects the whole certificates page. It is a bit dependent on the same css